package.json

```diff
@@ -42,6 +42,7 @@
   },
   "dependencies": {
     "bytes": "^3.1.0",
+    "ckan_ng_montreal_theme": "^1.0.1",
```

Comment:
@starsinmypockets not for now, but let's think about better naming for our themes, e.g., may be we can have just `frontend-montreal` so it is clear/shorter and we can have description/title where we mention it is a theme for NG CKAN. WDYT?